app/src/main/java/com/bps/gotwinter2021/ui/overview/OverviewViewModelFactory.kt

```diff
@@ -0,0 +1,20 @@
+package com.bps.gotwinter2021.ui.overview
+
+import android.app.Application
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.bps.gotwinter2021.favorites.database.FavoriteDatabaseDao
+
+class OverviewViewModelFactory(
+    private val passedName: String,
+    private val application: Application,
```

Comment:
You don't need to create a viewmodel factory when you can use the function in ViewModelUtils.kt